axonserver/src/main/java/io/axoniq/axonserver/exception/ErrorCode.java

```diff
@@ -48,6 +49,8 @@
     CANNOT_DELETE_INTERNAL_CONTEXT("AXONIQ-2304", Status.PERMISSION_DENIED, HttpStatus.FORBIDDEN, true),
     MAX_CLUSTER_SIZE_REACHED("AXONIQ-2305", Status.PERMISSION_DENIED, HttpStatus.FORBIDDEN, true),
     ALREADY_MEMBER_OF_CLUSTER("AXONIQ-2306", Status.ALREADY_EXISTS, HttpStatus.NOT_ACCEPTABLE, true),
+    REPLICATION_GROUP_UPDATE_IN_PROGRESS("AXONIQ-2107", Status.NOT_FOUND, HttpStatus.BAD_REQUEST, false),
+    REPLICATION_GROUP_CREATION_NOT_ALLOWED("AXONIQ-2306", Status.PERMISSION_DENIED, HttpStatus.FORBIDDEN, true),
```

Comment:
Duplicate code AXONIQ-2306, HttpStatus BAD_REQUEST (it is a user error), grpc Status.INVALID_ARGUMENT

Reply:
Remove CONTEXT_CREATION_NOT_ALLOWED